test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -210,9 +210,8 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, tf2_only, cp
 def run_data_service_test(ec2_connection, tensorflow_training, cmd):
     _, tensorflow_version = test_utils.get_framework_and_version_from_tag(tensorflow_training)
     ec2_connection.run(f"python -m pip install --upgrade pip")
-    ec2_connection.run(f"python -m pip install 'protobuf>=3.20,<3.21'")
     ec2_connection.run(f"python -m pip install tensorflow=={tensorflow_version}")
-    ec2_connection.run(f"python -m pip install 'protobuf>=3.20,<3.21'")
+    ec2_connection.run(f"python -m pip install 'protobuf >= 3.9.2,<3.20'")
```

Comment:
Same comment as above - Can we make this window smaller? The protobuf versions that seem to be actively supported recently are 3.16 and >=3.19.

Reply:
Updated